test/db/formats/mach0/coresymbolication

```diff
@@ -71,15 +71,15 @@ CMDS=<<EOF
 is*~SceneDelegate
 EOF
 EXPECT=<<EOF
-"f sym.__SceneDelegate_scene:willConnectToSession:options: 4 0x100006514"
-"f sym.__SceneDelegate_sceneDidDisconnect: 4 0x100006518"
-"f sym.__SceneDelegate_sceneDidBecomeActive: 4 0x10000651c"
-"f sym.__SceneDelegate_sceneWillResignActive: 4 0x100006520"
-"f sym.__SceneDelegate_sceneWillEnterForeground: 4 0x100006524"
-"f sym.__SceneDelegate_sceneDidEnterBackground: 4 0x100006528"
-"f sym.__SceneDelegate_window 16 0x10000652c"
-"f sym.__SceneDelegate_setWindow: 20 0x10000653c"
-"f sym.__SceneDelegate_.cxx_destruct 20 0x100006550"
+"f sym.-[SceneDelegate_scene:willConnectToSession:options:] 4 0x100006514"
```

Comment:
does this work properly?

Reply:
```
[i] ℤ rizin /bin/ls
[0x00006b60]> "f bla-[sdgsf] 4 @ 0x6b60"
[0x00006b60]> f~bal
[0x00006b60]> f~bla
0x00000000 4 bla__sdgsf
```

Reply:
So probably we should wait for this particular change until `f` ported to the newshell and maybe rest of the commands that work with flags.